src/fuse_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -114,153 +266,200 @@
+            name:       "SECRET_LC_UNLOCK_PARTITION",
             variant:    "Buffered",
             secret:     true,
             sw_digest:  false,
             hw_digest:  true,
             write_lock: "Digest",
             read_lock:  "Digest",
-            key_sel:    "Secret0Key",
+            key_sel:    "SecretLifeCycleUnlockKey",
             integrity:  true,
             bkout_type: false,
             items: [
                 {
-                    name: "TEST_UNLOCK_TOKEN",
-                    // This will generate a random default to be output in
-                    // case partition has not initialized or is in error state.
-                    // If not specified, a value of '0 will be used.
+                    name: "TEST_UNLOCK_TOKEN_0",
                     inv_default: "<random>",
                     size: "16"
-                }
+                },
                 {
-                    name: "TEST_EXIT_TOKEN",
+                    name: "TEST_UNLOCK_TOKEN_1",
                     inv_default: "<random>",
                     size: "16"
-                }
+                },
+                {
+                    name: "TEST_UNLOCK_TOKEN_2",
+                    inv_default: "<random>",
+                    size: "16"
+                },
+                {
+                    name: "TEST_UNLOCK_TOKEN_3",
+                    inv_default: "<random>",
+                    size: "16"
+                },
+                {
+                    name: "TEST_UNLOCK_TOKEN_4",
+                    inv_default: "<random>",
+                    size: "16"
+                },
+                {
+                    name: "TEST_UNLOCK_TOKEN_5",
+                    inv_default: "<random>",
+                    size: "16"
+                },
+                {
+                    name: "TEST_UNLOCK_TOKEN_6",
+                    inv_default: "<random>",
+                    size: "16"
+                },
+                {
+                    name: "TEST_UNLOCK_TOKEN_7",
+                    inv_default: "<random>",
+                    size: "16"
+                },
             ],
-            desc: '''Secret partition 0.
-            This contains TEST lifecycle unlock tokens.
+            desc: '''Secret life-cycle unlock token partition.
             '''
-        }
+        },
         {
-            name:       "SECRET1",
+            name:       "SECRET_LC_MANUF_PARTITION",
             variant:    "Buffered",
             secret:     true,
             sw_digest:  false,
             hw_digest:  true,
             write_lock: "Digest",
             read_lock:  "Digest",
-            key_sel:    "Secret1Key",
+            key_sel:    "SecretLifeCycleManufKey",
             integrity:  true,
             bkout_type: false,
             items: [
                 {
-                    name:   "UDS_SEED",
-                    size:   "48"
-                }
+                    name: "MANUF_TOKEN",
```

Comment:
nit: It may be good to clarify this a bit more. Is this an unlock token? If so, can we call it `MANUF_UNLOCK_TOKEN`? The same applies for all the tokens below. 

For cases where we have authorization tokens, it would be good to encode that in the name. 

This will make the programming interface a bit more ergonomic.